src/etc/rc.bootup

```diff
@@ -227,6 +229,9 @@ if (!$debugging) {
 /* re-make hosts file after configuring interfaces */
 system_hosts_generate();
 
+/* add configured trusted CA certificates to /etc/ssl/certs */
+ca_trustedcerts_generate();
+
```

Comment:
> Unless you can check to be sure the state is consistent and update as needed, it is necessary. 
Which is how it should be done.  Verify what is there against what config says should be there and add/remove accordingly.  Also include a "synchronize" button on the CA page for user to be able to manually initiated the same thing.